packages/modal/addon/styles/components/_nucleus-modal.scss

```diff
@@ -82,6 +81,9 @@ $slider-width: 600px;
       .title {
         font-size: $font-size-18;
         font-weight: $font-weight-600;
+        margin: 0;
+        line-height: 1.1;
+        -webkit-font-smoothing: antialiased;
```

Comment:
`-webkit-font-smoothing: antialiased;`

Why? I thought we weren't going to do font smoothing within nucleus but leave it up to the consuming apps to do it?

Reply:
Our Nucleus Documentation site has actually always applied `-webkit-font-smoothing` on our elements (through vendor.css). So what we assumed as our original font on screen was actually always a font-smoothed font. I figured, we eventually want our font what to look like what it looks on the Nucleus documentation site and for that we will have to apply `-webkit-font-smoothing` to everything!

Reply:
As discussed, can we use the CSS from @nucleus/core ?